rs-backend/src/git.rs

```diff
@@ -0,0 +1,59 @@
+use std::env;
+use std::path::Path;
+use std::process::Command;
+
+const CHECKOUT_OPTION: &'static str = "master";
+const GIT: &'static str = "git";
+const GIT_CHECKOUT: &'static str = "checkout";
+const GIT_PULL: &'static str = "pull";
+
+
+pub fn get_repo_path() -> Result<String, &'static str> {
+    if env::args().count() < 2 {
+        return Err("No argument supplied, needs location of data repo.");
+    }
+    Ok(env::args().nth(1).unwrap())
+}
+
+pub fn update_repo(repo_path: String) {
+    let working_dir = Path::new(&repo_path);
+
+    if checkout_master(working_dir).is_ok() {
+        if pull_updates(working_dir).is_ok() {
```

Comment:
Return a result from `update_repo` and make these `?`; that'll make it easier to handle any print any potential errors in more detail.